src/platform/packages/shared/response-ops/alerts-table/types.ts

```diff
@@ -148,6 +147,12 @@ export interface AlertsTableOnLoadedProps {
   totalAlertsCount: number;
 }
 
+export interface AlertsTableSortCombinations {
```

Comment:
We were using the `SortCombinations` type from the ES client for the `sort` prop (which is an alias for different formats), even though we were only extracting this specific format. [This](https://github.com/elastic/kibana/pull/234309/files#diff-17f37de95c4824afbd67e7d91afc70207a19c185f6a7af2742ff8699a2ae0ba2) sort in O11y for example was completely ignored by the underlying data grid and only sent to the query itself because of this. This new narrower type ensures we only receive the format we actually handle